sql/core/src/main/scala/org/apache/spark/sql/DataFrameReader.scala

```diff
@@ -298,6 +299,8 @@ class DataFrameReader private[sql](sparkSession: SparkSession) extends Logging {
    * <li>`timestampFormat` (default `yyyy-MM-dd'T'HH:mm:ss.SSSZZ`): sets the string that
    * indicates a timestamp format. Custom date formats follow the formats at
    * `java.text.SimpleDateFormat`. This applies to timestamp type.</li>
+   * <li>`timeZone` (default session local timezone): sets the string that indicates a timezone
```

Comment:
I'd like to use `timeZone` for the option key as the same as `spark.sql.session.timeZone` for config key for the session local timezone.
What do you think?